concourse/tasks/guest-package-build.yaml

```diff
@@ -0,0 +1,28 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: gcr.io/gcp-guest/daisy-builder
```

Comment:
it seems you could use gcr.io/compute-image-tools/daisy here, if you're not using any of the extra stuff in daisy-builder

Reply:
Yep. Fix this.